dev/integration_tests/new_gallery/lib/studies/reply/adaptive_nav.dart

```diff
@@ -574,16 +574,14 @@ class _MobileNavState extends State<_MobileNav> with TickerProviderStateMixin {
   }
 
   bool _handleScrollNotification(ScrollNotification notification) {
-    if (notification.depth == 0) {
-      if (notification is UserScrollNotification) {
-        switch (notification.direction) {
-          case ScrollDirection.forward:
-            _bottomAppBarController.forward();
-          case ScrollDirection.reverse:
-            _bottomAppBarController.reverse();
-          case ScrollDirection.idle:
-            break;
-        }
+    if (notification case UserScrollNotification(depth: 0)) {
```

Comment:
This one makes total sense to me!
So this means that for example

```dart
UserScrollNotification(depth:0, direction: ScrollDirection.forward) == UserScrollNotification(depth:0, direction: ScrollDirection.reverse)
```
in the if statement?